rjc_stock/models/stock_inventory.py

```diff
@@ -0,0 +1,10 @@
+from odoo import models, fields
+
+
+class Inventory(models.Model):
+    _name = 'stock.inventory'
+    _inherit = ['stock.inventory', 'mail.thread', 'mail.activity.mixin']
+
+    state = fields.Selection(
+        track_visibility="onchange",
+    )
```

Comment:
IMO, You should to add track_visibility="onchange" for other field too because if user change some field not state. User will not see log to change that field.

**Objective for logging**
1. We know that who are changing the field and what is value before change field.